tests/connector.test.js

```diff
@@ -0,0 +1,5 @@
+const psql = require('../db/connector.js');
+
+test('Expect the connector to return a query method', () => {
+  expect(psql).toHaveProperty('query');
+});
```

Comment:
You've committed the ultimate sin: you failed to put an empty line at the end of your file 😈 🤣